src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_nodejs/nodejs_translate.v2.json.baseline

```diff
@@ -67,21 +59,17 @@ translate.languages.list(request, function(err, response) {
 // 1. If not already done, enable the Translate API
 //    and check the quota for your project at
 //    https://console.developers.google.com/apis/api/translate
-// 2. This sample uses Application Default Credentials for authentication.
-//    If not already done, install the gcloud CLI from
-//    https://cloud.google.com/sdk and run
-//    `gcloud beta auth application-default login`
-// 3. Install the Node.js client library by running
+// 2. Install the Node.js client library by running
 //    `npm install googleapis --save`
 
 var google = require('googleapis');
 var translate = google.translate('v2');
 
-var apiKey = ''; // TODO: Update placeholder with desired API key.
+var apiKey = '';  // TODO: Update placeholder with desired API key.
 
 var request = {
   // The text to translate
-  q: [],  // TODO: Update placeholder value.
+  q: '',  // TODO: Update placeholder value.
```

Comment:
(compilecheck passes, so I figure I forgot to update some baselines when I submitted the override and int64 fixes)